test/docker-compose.yml

```diff
@@ -22,19 +22,19 @@ services:
 
   # Parachain
   parachain:
-    image: paritytech/ci-linux:production
-    working_dir: /var/www/artemis-node
+    image: mangata/mangata-node:latest
```

Comment:
Good job adapting this for mangata! Have you run the tests against that setup?

Reply:
Thank you.
All the tests were run using this setup. You do have to build this image using the DockerFile in /setup